### PR TITLE
New weather station & telemtry field

### PR DESCRIPTION
**Pull request by adelin-diac**
@kevin-hereworks

### CODE REVIEW

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -461,6 +461,18 @@ literals:
 - outside_air_pm10pt0_density_sensor:
     flexible_min: 0.0
     flexible_max: 0.001
+- outside_air_pm1pt0_density_sensor:
+    flexible_min: 0.0
+    flexible_max: 0.001
+- outside_air_no_density_sensor:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:467]: Can you clarify why this field is useful on a weather station?

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 2]
```diff
@@ -461,6 +461,18 @@ literals:
...
+    flexible_min: 0.0
+    flexible_max: 0.0002
+- outside_air_no2_density_sensor:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:470]: Same

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 3]
```diff
@@ -461,6 +461,18 @@ literals:
...
+    flexible_min: 0.0
+    flexible_max: 0.0002
+- outside_air_so2_density_sensor:
```
Comment (shambergoldstein) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:473]: Same

**Review by shambergoldstein — CHANGES_REQUESTED** [review]
See comments on field usefulness.

**Comment by adelin-diac** [comment]
Weather station is the name of device given to us with corresponding points being measured. Would it make more sense to change this to a `SENSOR` type? I've updated to `SENSOR` for now.

**Comment by cstirdivant** [comment]
@kevin-hereworks  @adelin-diac if this is supposed to model "add on" functionality of a weather station I would generate a new type in `WEATHER`.

**Comment by adelin-diac** [comment]
reverted to `WEATHER` type and left comment above on why those fields are included.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -6620,3 +6620,19 @@ VMADC:
...
+  description: "Outside air quality monitoring (particulates and gases)"
+  is_abstract: true
+  uses:
+  - outside_air_pm1pt0_density_sensor
+  - outside_air_pm2pt5_density_sensor
+  - outside_air_pm10pt0_density_sensor
+  - outside_air_no_density_sensor
+  - outside_air_no2_density_sensor
+  - outside_air_so2_density_sensor
+  - outside_air_co_concentration_sensor
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6634]: You are mixing `density` and `concentration` measurements; please confirm the units of measurement for each sensor.

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 4 above)
Comment (adelin-diac) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6634]: we are being informed that the measurements for `NO`, `NO2`, `SO2` are in `micrograms per cubic meters` which is also what we see in telemetry message. `CO` is "ppm" so I used concentration for this field.

Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:6634]: Thank you for confirming!

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**